src/components/panel/panel.spec.js

```diff
@@ -105,6 +108,62 @@ describe('$mdPanel', function() {
     expect(PANEL_EL).not.toExist();
   });
 
+  it('should call onDomAdded if provided when adding the panel to the DOM', function() {
+    var onDomAdded = jasmine.createSpy();
+    var animationHookConfig = angular.extend({'onDomAdded': onDomAdded}, DEFAULT_CONFIG);
+
+    panelRef = $mdPanel.create(animationHookConfig);
+
+    panelRef.attachOnly().then(function() {
```

Comment:
These thens never get called since this isn't an async test. You can test that by adding a `fail` before the expects -- the test will still pass. The close function already clears the promise queue before continuing (via $scope.$apply and flush). For attach, you should add a function below, like `closePanel`, `openPanel`, etc.